app/jsconfig.json

```diff
@@ -3,6 +3,7 @@
     "target": "es2017",
     "allowSyntheticDefaultImports": false,
     "baseUrl": "./",
+    "moduleResolution": "node",
```

Comment:
Are you able to peek definitions after this change ?

Reply:
yup